app/assets/stylesheets/minimal.scss

```diff
@@ -37,6 +37,9 @@
 @import 'typography';
 @import 'buttons';
 @import 'widgets';
+@import 'forms';
+@import 'modal';
+@import 'notice';
```

Comment:
I'd be careful with importing new files unless you're 100% sure they gonna be used globally. Naming of these files indicate they should be used globally, but with the `forms.scss` example I'd double check that :).

Reply:
Since my branch has the new "Crayons" components & styles, I think that I'll actually remove some of these repetitions from my branch here and implement "Crayons"!

Reply:
Just keep in mind, crayons is still "in beta" :D I'd be careful with using it globally. just FYI :)

Reply:
@ludwiczakpawel thanks!

Reply:
> I put in the same reviews as I put in [previous PR](https://github.com/thepracticaldev/dev.to/pull/6410) - looks like these still need to be addressed.
> 
> Also, I haven't checked this current PR locally because I'm having some issues with my `gh` client, but the previous one had troubles with handling responsiveness too.

Ahh good point. I had gotten rid of `notice.scss` but if I'm keeping `forms.scss`, I should make it more local to onboarding. I'll add the parent class tags around it to keep it local.